test/output_schema_test.py

```diff
@@ -337,50 +308,6 @@ def test_change_existing_schema(self):
 
         from socrata.http import get
 
-        (ok, result) = get("https://{domain}/id/{fourfour}".format(domain = auth.domain, fourfour = rev.attributes['fourfour']), auth)
-        assert ok, result
+        result = get("https://{domain}/id/{fourfour}".format(domain = auth.domain, fourfour = rev.attributes['fourfour']), auth)
 
         self.assertEqual(result, [{'a': '1', 'd': '6'}, {'a': '2', 'd': '7'}, {'a': '3', 'd': '8'}, {'a': '4', 'd': '9'}])
-
-
-    # Broken due to ingress strategy flags - if we leave this long enough maybe
-    # the OBE will be gone and we can just delete it
```

Comment:
it's great to see a plan sorta kinda come to fruition

Reply:
wooooo